### DISCUSSION TITLE
Suggestion: Daeyalt essence mining timer

### DISCUSSION THREAD

**Discussion by ItsVendum** [runelite/runelite#16588] [Ideas]
Daeyalt essence plugin with the following features:
- Highlight the active Daeyalt essence rock.
- A counter that shows when the next rock becomes active
- _Optional:_ Notification when the next rock becomes active

**Comment by Noriomusha** [comment]
With the mining plugin turned on it does show the current rock with timer along with what rock has the next most time. See screenshot. 
![image](https://github.com/runelite/runelite/assets/7958905/8c2d8fcb-2a1f-4cc9-ba23-c44ad968d735)
